main.py

```diff
@@ -264,6 +268,7 @@ def _store_changed_flows(self, command, flow, switch):
         installed_flow = {}
         installed_flow['command'] = command
         installed_flow['flow'] = flow
+        installed_flow['timestamp'] = time.time()
```

Comment:
I wonder if we could use `created_at` like [storehouse](https://github.com/kytos-ng/storehouse/blob/master/main.py#L41), that way, we could always use the same key for referring to a timestamp when that resource was created, facilitation one day if we also indexin the DB  by `created_at`. Wdyt?

Reply:
I agree with @viniarck, `created_at` is more meaningful.

Reply:
Thank you @viniarck and @ajoaoff, you were right and I've addressed this change on commit 6fdad44